### PR TITLE
[DEV-834]: Instrument Kafka Produce/Consume non Datadog use-case

### PR DESCRIPTION
**Pull request by br-kearns**
# Description

- Added instrumentation tracing with OTEL for non-datadog kafka use-cases
- Added Strimzi/Kafka yaml configuration files for local Kafka cluster setup
- Added documentation readme files
- Added shell scripts to manage create/delete of local kafka cluster resources

JIRA: [DEV-834]

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation has been updated

[DEV-834]: https://causely.atlassian.net/browse/DEV-834?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: helm/video/values.yaml [hunk 1]
```diff
@@ -173,23 +196,6 @@ services:
...
-      password: "" # TODO
-      tls_enable: true
-      sasl_enable: true
-
-  - name: "message-broker-producer"
-    type: rabbitmq-producer
-    config:
-      tracing_service_name: message-broker
-      url: "amqp://guest:guest@message-broker:5672/"
-
```
Comment (br-kearns) [inline helm/video/values.yaml]: Just a note here on why I removed this:

The `Script: | ...` body of the `background_services:` for "encoder-data-broker-consumer" relies on
```
ctx.get_service("data-broker-producer");
```

so having two separate named definitions for producers (e.g. for Rabbit-MQ vs Kafka) means we would also need to remember to modify that `get_service` call depending on which stack we are using. It is simpler to just modify the configuration details for the particular type of producer we are using under the same service name.

File: helm/video/values.yaml [hunk 2]
```diff
@@ -98,11 +98,17 @@ background_services:
   # - It consumes messages from the message broker (the video status, send by the encoder)
   # - Updates the video status in the upload database
   - name: "upload-message-broker-consumer"
-    type: rabbitmq-consumer
```
Comment (ib-steffen) [inline helm/video/values.yaml]: why change the scenario?

File: helm/video/values.yaml (hunk 1 above)
Comment (ib-steffen) [reply helm/video/values.yaml]: The goal of this architecture was to have rabbitmq and kafka for different purposes. Please keep the original architecture.

File: helm/video/values.yaml (hunk 2 above)
Comment (br-kearns) [reply helm/video/values.yaml]: After discussion, I added back rabbitmq aspect

File: helm/video/values.yaml (hunk 1 above)
Comment (br-kearns) [reply helm/video/values.yaml]: rolled back

**Review by esara — APPROVED** [review]

**Pull request merged**